server/repository/src/mock/name_store_join.rs

```diff
@@ -2,16 +2,6 @@ use crate::NameStoreJoinRow;
 
 use super::{mock_name_a, mock_name_store_a, mock_name_store_b, program_master_list_store};
 
-pub fn mock_name_store_join_a() -> NameStoreJoinRow {
```

Comment:
Invalid mock data ;) seems no test was relying on it.

Reply:
why was it invalid? or just unused?

Nit: mock names start with mock_name_store_join_b() now...

Reply:
I vote for rename so it starts with a()~

Reply:
Not sure what rename is being requested?

The reason it's invalid is because stores shouldn't have an NSJ that points to itself. Least a store becomes a customer/supplier of itself.

Reply:
rename join_b -> join_a() so it does starts in order D:

Reply:
68f4266343ea deal with it lol 🤣

Reply:
my final offer is to just name them "join_1()" and so on lol.